packages/andive/src/components/menu.js

```diff
@@ -80,9 +80,10 @@ const OptionGroupLayout = styled.div`
   cursor: pointer;
 `
 
-function OptionGroup({children, label, onClick, ...props}) {
+function OptionGroup({children, label, onClick, leftIcon, color, ...props}) {
```

Comment:
If you introduce a color prop, I would like to color the whole line, so:
- apply it to Info.Label (Change Info > Info.Label with Box > Typography.Body1)
- apply it to ArrowRightIcon

Also, fallback on palette.mediumPrimary in the prop definition (..., color = palette.mediumPrimary, ...)